src/js/components/helptips/__snapshots__/onboardingcompletetip.test.js.snap

```diff
@@ -76,7 +76,7 @@ exports[`renders correctly 1`] = `
     What next?
     <div>
       <a
-        href="https://docs.mender.io//getting-started/deploy-to-physical-devices#prepare-the-disk-image"
+        href="https://docs.mender.io/getting-started/deploy-a-system-update-demo"
```

Comment:
This should now be https://docs.mender.io/getting-started/on-premise-installation/deploy-a-system-update-demo
The page URL changed since this PR was made, sorry!